build-test.xml

```diff
@@ -16284,8 +16295,11 @@ database.type=postgresql
 			<then>
 				<var name="database.sqlserver.docker.image" value="mcr.microsoft.com/mssql/server:2017-CU19-ubuntu-16.04" />
 			</then>
+			<elseif>
+				<equals arg1="${database.version}" arg2="2022" />
+			</elseif>
 			<else>
-				<var name="database.sqlserver.docker.image" value="mcr.microsoft.com/mssql/server:2019-GA-ubuntu-16.04" />
+				<var name="database.sqlserver.docker.image" value="mcr.microsoft.com/mssql/server:2022-RTM-CU2-ubuntu-20.04" />
```

Comment:
@CharlotteFrancis There is definitely something wrong here. I'm guessing it should be:

```
			<elseif>
				<equals arg1="${database.version}" arg2="2022" />
				<then>
					<var name="database.sqlserver.docker.image" value="mcr.microsoft.com/mssql/server:2022-RTM-CU2-ubuntu-20.04" />
				</then>
			</elseif>
```